### PR TITLE
F/added subtext status informations

### LINKED ISSUE #480: Provide more details to "Current MAC address" field in "Status information"
There were people that thought they'd need to enter their phones MAC address in order to be able to use the WLAN.  I can understand this confusion to some extend and think we should provide more details next to the field as well as the form for changing.

(thanks @jnsga)

### PR DESCRIPTION
**Pull request by agmes4**
You're opening a pull request? __Awesome!__

Please make sure though you did the following (tick off everything you
already did):

- [x] Run the tests and see them pass
- [x] Rebase your branch on top of `develop`
- [x] Include tests for features you introduced / bugs you fixed

This issue fixes #480, #468

### CODE REVIEW

**Comment by agmes4** [comment]
some body should definitively look into the translation files for spelling and stuff

**Comment by agmes4** [comment]
I  have now added a test for the last_received_update property for more I would need a lib for freezing the time like freezegun or so

File: sipa/model/user.py [hunk 1]
```diff
@@ -152,7 +152,12 @@ def traffic_history(self) -> list[dict]:
 
     def generate_rows(self, description_dict: dict) -> t.Iterator[Row]:
         for key, val in description_dict.items():
-            yield Row(description=val, property=getattr(self, key))
+            yield Row(property=getattr(self, key), **self.__text_to_dict(val))
+
+    def __text_to_dict(self, val: str|list[str]) -> dict:
+        if isinstance(val, list):
+            return {'description': val[0], 'subtext': val[1] if len(val) == 2 else val}
+        return {'description':val, 'subtext': None}
```
Comment (lukasjuhrich) [inline sipa/model/user.py:160]: Please do not use two underscores unless you know what they do! They have special behaviour (mangling).
I would call this `_parse_description` instead.
A `match` like this (warning: untested) might be easier to read:
```py
match val:
    case [d, s]:
        return {"description": d, "subtext": s}
    case [d]:
        return {"description": d, "subtext": None}
    return {"description": val, "subtext": None}

File: sipa/model/user.py [hunk 2]
```diff
@@ -27,7 +27,7 @@ class AuthenticatedUserMixin:
     is_anonymous = False
 
 
-Row = namedtuple('Row', ['description', 'property'])
+Row = namedtuple('Row', [ 'property','description', 'subtext'])
```
Comment (lukasjuhrich) [inline sipa/model/user.py:30]: You can use a [`typing.NamedTuple`](https://docs.python.org/3/library/typing.html#typing.NamedTuple) with the advantage that you can specify a default Value of `None` for the `subtext`, then you don't have to add that to the dictionary every time in the parsing method below

File: sipa/translations/de/LC_MESSAGES/messages.po [hunk 3]
```diff
@@ -1059,6 +1059,11 @@ msgstr ""
 msgid "Ausgehend"
 msgstr ""
 
+msgid "Eingeangenezahlung"
```
Comment (lukasjuhrich) [inline sipa/translations/de/LC_MESSAGES/messages.po:1062]:
```suggestion
msgid "Eingegangene Zahlung"
```

File: sipa/blueprints/usersuite.py [hunk 4]
```diff
@@ -57,28 +57,30 @@ def index():
...
+        ('mac', [gettext("Aktuelle MAC-Adresse"), gettext("die MAC Adresse des Anschlusses des per Kabel verbundenen Gerätes")]),
         ('mail', gettext("E-Mail-Adresse")),
         ('mail_confirmed', gettext("Status deiner E-Mail-Adresse")),
         ('mail_forwarded', gettext("E-Mail-Weiterleitung")),
         ('wifi_password', gettext("WLAN Passwort")),
         # ('hostname', gettext("Hostname")),
         # ('hostalias', gettext("Hostalias")),
         ('userdb_status', gettext("MySQL Datenbank")),
-        ('finance_balance', gettext("Kontostand") + finance_update_string),
+        ('finance_balance', [gettext("Kontostand") + finance_update_string, gettext("Eingeangenezahlung") + ": " + finance_received_string]),
```
Comment (lukasjuhrich) [inline sipa/blueprints/usersuite.py:83]:
```suggestion
        ('finance_balance', [gettext("Kontostand") + finance_update_string, gettext("Eingegangene Zahlung") + ": " + finance_received_string]),
```

File: sipa/translations/en/LC_MESSAGES/messages.po [hunk 5]
```diff
@@ -1101,6 +1101,12 @@ msgstr "Incoming"
 msgid "Ausgehend"
 msgstr "Outgoing"
 
+msgid "Eingeangenezahlung"
```
Comment (lukasjuhrich) [inline sipa/translations/en/LC_MESSAGES/messages.po:1104]:
```suggestion
msgid "Eingegangene Zahlung"
```

File: tests/model/test_finance.py [hunk 6]
```diff
@@ -51,3 +55,18 @@ def test_balance_not_editable(self, balance):
...
+        raw_balance = 30
+        history = []
+        last_update = datetime.date.today()
+
+    @pytest.fixture(scope="class")
+    def last_recv(self) -> ActiveProperty:
+        return self.StaticFinanceInformation().last_received_update
+
+    def test_has_correct_balance(self, last_recv):
+        assert last_recv != datetime.date.today()
```
Comment (lukasjuhrich) [inline tests/model/test_finance.py:72]: How would you expand on this test with a library like `freezegun`?

**Review by lukasjuhrich — CHANGES_REQUESTED** [review]

**Comment by lukasjuhrich** [comment]
So sorry for the long delay! Would you care to `rebase --interactive --autosquash` your branch, or should I?

File: sipa/model/user.py (hunk 2 above)
Comment (agmes4) [reply sipa/model/user.py]: thanks thats great

File: sipa/model/user.py (hunk 1 above)
Comment (agmes4) [reply sipa/model/user.py]: should be resolved now

**Comment by agmes4** [comment]
I will rebase tomorrow cause I am getting tired. But thanks for your review ;)

**Comment by agmes4** [comment]
branch was rebased

**Comment by lukasjuhrich** [comment]
Currently it looks like this:
image

Content-wise I think this is a good improvement, I'll just check why the translation does not work, then merge.

**Pull request merged**